modules/aws_asg/main.tf

```diff
@@ -265,11 +265,15 @@ resource "aws_autoscaling_group" "workers" {
 }
 
 resource "aws_autoscaling_policy" "workers_remove_capacity" {
-  name                   = "${var.env}-${var.index}-workers-${var.site}-remove-capacity"
-  scaling_adjustment     = "${var.worker_asg_scale_in_qty}"
-  adjustment_type        = "ChangeInCapacity"
-  cooldown               = "${var.worker_asg_scale_in_cooldown}"
-  autoscaling_group_name = "${aws_autoscaling_group.workers.name}"
+  name                      = "${var.env}-${var.index}-workers-${var.site}-remove-capacity"
+  adjustment_type           = "ChangeInCapacity"
+  policy_type               = "StepScaling"
+  autoscaling_group_name    = "${aws_autoscaling_group.workers.name}"
+  estimated_instance_warmup = "${var.worker_asg_scale_in_cooldown}"
+
+  step_adjustment {
+    scaling_adjustment = "${var.worker_asg_scale_in_qty}"
+  }
```

Comment:
IIUC we need to specify multiple `step_adjustment` blocks in order to take advantage of the stepping, no?  ...which makes sense to me given the AWS blog example and https://www.terraform.io/docs/providers/aws/r/appautoscaling_policy.html#step_adjustment